dashboard/fixtures/certificates.json

```diff
@@ -5,14 +5,14 @@
 	    "certificate_type": "verified",
 	    "status": "downloadable",
 	    "download_url": "http://www.example.com/demo.pdf",
-	    "grade": "0.98"
+	    "grade": 98
```

Comment:
i don't like this either, but the method where we post-process data coming from edx is being mocked out in multiple places: https://github.com/mitodl/micromasters/blob/master/dashboard/api_test.py#L878. `get_student_certificates` and `get_student_enrollments` are doing _way_ too many things right now. i have an [issue open](https://github.com/mitodl/micromasters/issues/743) to refactor the code for CachedEnrollments/Certificates, which would include those methods. I can make a note to revert this JSON file to mimic edx data again